manifests/kustomize/base/installs/generic/pipeline-install-config.yaml

```diff
@@ -17,14 +17,16 @@ data:
   dbType: mysql
   mysqlHost: mysql
   mysqlPort: "3306"
+  postgreHost: postgres-service
+  postgrePort: "5432"
```

Comment:
since we are generating a new configmap (e.g.: `manifests/kustomize/env/platform-agnostic-postgresql/kustomization.yaml`), do we still need this here? can it be removed?